Mathlib/Tactic/NormNum/Core.lean

```diff
@@ -215,55 +216,6 @@ mutual
     (·.1) <$> Simp.main e ctx (methods := methods)
 end
 
--- FIXME: had to inline a bunch of stuff from `simpGoal` here
-/--
-The core of `norm_num` as a tactic in `MetaM`.
-
-* `g`: The goal to simplify
-* `ctx`: The simp context, constructed by `mkSimpContext` and
-  containing any additional simp rules we want to use
-* `fvarIdsToSimp`: The selected set of hypotheses used in the location argument
-* `simplifyTarget`: true if the target is selected in the location argument
-* `useSimp`: true if we used `norm_num` instead of `norm_num1`
--/
-def normNumAt (g : MVarId) (ctx : Simp.Context) (fvarIdsToSimp : Array FVarId)
```

Comment:
This change looks broadly fine, but I don't feel confident signing off on all of it. Another pair of eyes would be appreciated.